schema/viewer.js

```diff
@@ -28,13 +28,17 @@
  * additional fields below.
  */
 
+import CausalityJWT from './causality_jwt';
+import Me from './me';
 import Sales from './sales';
 import { GraphQLObjectType } from 'graphql';
 
 const ViewerType = new GraphQLObjectType({
   name: 'Viewer',
   description: 'A Viewer',
   fields: () => ({
+    causality_jwt: CausalityJWT,
```

Comment:
CausalityJWT as it is used takes two params (`mode`, `saleId`) and the specific query that we need to make is 
`
query {
  causality_jwt(role: OPERATOR, sale_id: "some-sale-id")
  me {
      paddle_number
      type
      bidders(sale_id: "${saleId}") {
        id
        qualified_for_bidding
      }
    }
}
```

Which can be wrapped like
```
query {
  viewer {
    causality_jwt(role: OPERATOR, sale_id: "some-sale-id")
    me {
      paddle_number
      type
      bidders(sale_id: "${saleId}") {
        id
        qualified_for_bidding
      }
    }
  }
}
```

Since a "Viewer" is effectively a wildcard, any kind of type can go in there (as it is commonly used). See this proposed sketch as an example: https://github.com/artsy/metaphysics/pull/531#issuecomment-275872962